crates/xline/src/storage/lease_store/mod.rs

```diff
@@ -52,6 +54,7 @@ pub(crate) struct LeaseStore {
     lease_collection: Arc<LeaseCollection>,
     /// Db to store lease
     db: Arc<DB>,
+    #[allow(unused)] // used in tests
```

Comment:
Does this index only used in tests? Can we use `cfg` to instead?

Reply:
Fixed. I completely removed the index from the struct and construct it in tests.